index.html

```diff
@@ -10,346 +10,308 @@
+        </h1>
         <p>
-          My name is Dagmawi Yohannes I'm a Front End Engineer based in
-          Addis Ababa, Ethiopia ☀️. I describe myself as a passionate
-          developer who loves coding, open source, and the web platform
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
         </p>
-        <br />
-      </section> 
-      <div class="h2"></div>
-      <div class="social-links">
-        <ul class="ulist">
-          <li class="ulist-icon">
-            <a href="https://twitter.com/dag_mawi111 "
-              ><img src="./images/twitter.svg" alt="twitter icon"
-            /></a>
+        <ul class="social-media">
+          <li>
+            <a href="https://twitter.com" target="_blank">
+              <img src="./assets/social-media/twitter.svg" alt="twitter icon" />
+            </a>
           </li>
-          <li class="ulist-icon">
-            <a href="https://linkedin.com/in/dagmawi-yohannes-b36488236"
-                ><img src="./images/Linkedinicon.svg" alt="linkedin icon"
-            /></a>
+          <li>
+            <a href="https://www.linkedin.com" target="_blank">
+              <img
+                src="./assets/social-media/linkedin.svg"
+                alt="linkedin icon"
+              />
+            </a>
           </li>
-          <li class="ulist-icon">
-            <a href="https://medium.com/dagmawi.yoha03"
-                ><img src="./images/medium.svg" alt="medium icon"
-            /></a>
+          <li>
+            <a href="https://medium.com/@dagmawi" target="_blank">
+              <img src="./assets/social-media/medium.svg" alt="medium icon" />
+            </a>
           </li>
-          <li class="ulist-icon">
-            <a href="https://github.com/dagmawiii03"
-                  ><img src="./images/github.svg" alt="github icon"
-            /></a>
+          <li>
+            <a href="https://github.com" target="_blank">
+              <img src="./assets/social-media/github.svg" alt="github icon" />
+            </a>
           </li>
-        </ul>
-      </section>
-      <section class="pointer">
-        <ul>
           <li>
-            <a href=""><i class="fas fa-chevron-down white-color"></i></a>
+            <a href="https://angel.co" target="_blank">
+              <img
+                src="./assets/social-media/angellist.svg"
+                alt="angellist icon"
+              />
+            </a>
           </li>
         </ul>
+        <button type="button">
+          <img src="./assets/chr-btn.svg" alt="arrow down button" />
+        </button>
       </section>
-      <!--landing pages-->
-      <section class="pages">
-        <section class="header">
-          <h3>
-            My Recent <br />
-            Works
-          </h3>
+    </section>
+    <section class="recent-works">
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 add `id="recent-works"` to allow the link in the navbar to go to the corresponding section for a better UX. Also for the about and contact sections.